scripts/sql/user_hssc_etl_create.sql

```diff
@@ -0,0 +1,25 @@
+/*
+ * After creating HSSC_ETL user, log in and create the private MPI DB Link:
+ 
+ CREATE DATABASE LINK "MPI"
+ CONNECT TO "PATIENT" IDENTIFIED BY <password>
+ USING 'hssc-cdw-mpidb-d:1521/mpidev'
+ ;
+ 
+ TODO: Orchestrate creation of user with grants and creation of private DB
+       link, which requires directly connecting as the new user.
+ */
+create user HSSC_ETL identified by changeme;
```

Comment:
can this be factored out into a procedure with HSSC_ETL as an argument to the procedure?